Frameworks/UIKit/UIButton.mm

```diff
@@ -552,8 +552,7 @@ - (void)setTitleColor:(UIColor*)color forState:(UIControlState)state {
     // ConvertUIColorToWUColor:nil creates a valid WUColor with null comObj
     // which isn't what we want
     if (color) {
-        WUColor* convertedColor = XamlUtilities::ConvertUIColorToWUColor(color);
-        WUXMSolidColorBrush* titleColorBrush = [WUXMSolidColorBrush makeInstanceWithColor:convertedColor];
+        WUXMSolidColorBrush* titleColorBrush = [[WUXMSolidColorBrush makeInstanceWithColor:XamlUtilities::ConvertUIColorToWUColor(color)] autorelease];
         if (titleColorBrush) {
             _states[state].inspectableTitleColor = [titleColorBrush comObj];
```

Comment:
If this becomes a hot path, we can pull in WRL.  

---
In reply to: [97687732](https://github.com/Microsoft/WinObjC/pull/1803#discussion_r97687732) [](ancestors = 97687732)